prototool.yaml

```diff
@@ -0,0 +1,51 @@
+# Paths to exclude when searching for Protobuf files.
+excludes:
+- vendor
+
+# Protoc directives.
+protoc:
+  # The Protobuf version to use from https://github.com/protocolbuffers/protobuf/releases.
+  # You probably want to set this to make your builds completely reproducible.
+  version: 3.7.1
+
+  # Additional paths to include with -I to protoc.
+  # By default, the directory of the config file is included,
+  # or the current directory if there is no config file.
+  includes:
+  - ../../googleapis/googleapis
```

Comment:
I'd recommend making this self-contained, perhaps using the `vendor/github.com/googleapis/googleapis` if available, especially if you want to use the breaking change detector down the road using git clones

Reply:
At the moment we vendor as little possible to reduce the maintenance burden of keeping vendor up-to-date. 

This seems like a good use case for go modules #1400